src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -2552,6 +2554,9 @@ void TemplateTable::jvmti_post_field_access(Register cache, Register index,
 
     __ load_field_entry(c_rarg2, index);
 
+    // Must prevent reordering of the following cp cache loads with bytecode load
+  __ membar(MacroAssembler::LoadLoad);
+
```

Comment:
I'm wondering if this can be in load_field_entry at the end so we don't miss any callers.  It might be a bit redundant with the ldar in the resolve_cache_and_index_for_field(), but that's for only the first time the field is resolved and in the interpreter, should not be an issue for performance.